doc/src/reference/api.md

```diff
@@ -0,0 +1,3 @@
+# API
```

Comment:
It's a little bit weird to have a whole section for a single badge and no text. Maybe put this together with the other badges on the overview page?

Reply:
there is already a docs.rs badge in README, however it's not very visible.
i wanted to give a quick link for anyone having the book opened